### PR TITLE
feat: use terawhiskers and add accent support

### PR DESCRIPTION
**Pull request by Anomalocaridid**
Replaces the old handlebars template with a new tera template and a justfile. Also adds accent color support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,8 @@
 1. Open your `fuzzel.ini` file, found in `$XDG_CONFIG_HOME/fuzzel/`
 (defaulting to `$HOME/.config/fuzzel/fuzzel.ini` if `$XDG_CONFIG_HOME` is unset).
 1. Remove existing `[color]` section if there is one.
-1. Insert the contents of one of the theme files in `./themes`
+1. Choose a flavor directory
+1. Insert the contents of one of the theme files in the flavor directory
```
Comment (uncenter) [inline README.md]:
```suggestion
1. Insert the contents of the flavor and accent combination of your choice from [`themes/`](./themes/) into your configuration file.
```

**Review by uncenter — APPROVED** [review]

**Comment by uncenter** [comment]
Looks great :) nice work

**Comment by Anomalocaridid** [comment]
I have updated the template to use `whiskers` 2.3.0 so that it does not have to explicitly set the palette.

**Comment by Anomalocaridid** [comment]
Thank you!

**Pull request merged**